src/components/MultiHopTrade/components/SharedTradeInput/SharedTradeInputFooter/SharedTradeInputFooter.tsx

```diff
@@ -87,6 +90,22 @@ export const SharedTradeInputFooter = ({
     return <Text translation={quoteStatusTranslation} />
   }, [quoteStatusTranslation])
 
+  // Calculate the delta percentage between the limit price and market price
+  const deltaPercentage = useMemo(() => {
```

Comment:
This doesn't seem to be actually used?

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/b6d10f93-fed4-4514-93ec-18695763c092" />

Reply:
As seen in private, this PR was missing a bunch of stuff.. should be done in [c748d59](https://github.com/shapeshift/web/pull/8970/commits/c748d59c6e8b3426ba1f33d79471bb6e7237a97a) I guess

Reply:
Confirmed now used!